src/csrf.zig

```diff
@@ -158,17 +158,30 @@ pub fn verify(options: VerifyOptions) bool {
     if (decoded.len < 64) {
         return false;
     }
+    // We successfully decoded the token but it could be a bad token
+    // base64 and hex can have ambiguity so we need to check for weird cases and reject them
+    // it could also be a handcrafted token that is invalid
 
     // Extract timestamp (first 8 bytes)
     const timestamp = std.mem.readInt(u64, decoded[0..8], .big);
 
     // Check if token has expired
     const current_time = @as(u64, @bitCast(std.time.milliTimestamp()));
+    if (current_time < timestamp and timestamp - current_time > 5 * 60 * 1000) {
```

Comment:
Is there a link to a popular implementation that does something similar? Would be good to have a sanity check that this makes sense to do.